packages/react/src/components/SuiteHeader/SuiteHeader.jsx

```diff
@@ -441,10 +449,15 @@ const SuiteHeader = ({
                       'data-testid': 'suite-header-help--about',
                       href: 'javascript:void(0)',
                       title: mergedI18N.about,
-                      onClick: async () => {
+                      onClick: async (e) => {
```

Comment:
nit: There is code duplication for the onClick functions and it can probably be refactored into a separate function that handles the logic of correctly opening a route.

Reply:
Good call. I extracted this out, and made the tests a little more robust by using different hrefs for all the routes to confirm the correct route was being called and not an accident since they were all just ibm.com.